psqlgraph/node.py

```diff
@@ -209,10 +210,15 @@ def __init__(self, node_id=None, properties=None, acl=None,
         self._props = {}
         self.system_annotations = system_annotations or {}
         self.acl = acl or []
-        self.properties = properties or {}
+        self.properties = self._defaults
+        self.properties.update(properties)
         self.properties.update(kwargs)
         self.node_id = node_id
 
+    @property
+    def _defaults(self):
```

Comment:
In Python a class attribute that starts with `_` usually indicates that this is a "private" class attribute. Is that the case? It seems like something the client would want to access.